themes/bootstrap3/templates/Related/Bookplate.phtml

```diff
@@ -0,0 +1,17 @@
+<?php $bookplates = $this->related->getBookplateDetails(); ?>
+<div id="bookplates">
+  <?php foreach($bookplates as $bookplate): ?>
+    <div class="bookplate">
+      <div class="bookplate-img">
+        <a href="<?=$bookplate['fullUrl']?>">
```

Comment:
This template needs escaping added everywhere -- i.e. `$this->escapeHtmlAttr()` for values inside HTML attributes, and `$this->escapeHtml()` for the title displayed inline. Otherwise, there's a risk that special characters (`&`, `<`, etc.) might garble the HTML output.

Reply:
Also, should we account for the possibility of only one URL instead of two? Should it be possible to only specify a full URL, or to have a thumbnail without a full URL? If only one image size is available, how should the code behave?

Reply:
Great suggestion! I fixed the escaping issue and set it up such that if only thumbnail or full URL is present, it will just put an image in the template. If both are present, it's a thumbnail and a link to the full image. I agree, this is more versatile. We might even use just one of them ourselves, since, that was what I was originally thinking.